HelpSource/Classes/SinOsc.schelp

```diff
@@ -13,51 +13,404 @@ Technically, code::SinOsc:: uses the same implementation as  link::Classes/Osc::
 
 subsection:: Other sinewaves oscillators
 
-LIST::
-## link::Classes/FSinOsc:: – fast sinewave oscillator
-## link::Classes/SinOscFB:: – sinewave with phase feedback
-## link::Classes/PMOsc:: – phase modulation sine oscillator
-## link::Classes/Klang:: – bank of sinewave oscillators
-## link::Classes/DynKlang:: – modulable bank of sinewave oscillators
-::
+ LIST::
+ ## link::Classes/FSinOsc:: – fast sinewave oscillator
+ ## link::Classes/SinOscFB:: – sinewave with phase feedback
+ ## link::Classes/PMOsc:: – phase modulation sine oscillator
+ ## link::Classes/Klang:: – bank of sinewave oscillators
+ ## link::Classes/DynKlang:: – modulable bank of sinewave oscillators
+ ::
+
+This is the same as  link::Classes/Osc::  except that the table is
+a sine table of 8192 entries.
+
+From Osc Supercollider Documentation Description:
```

Comment:
I don't see the point in copy-pasting the Osc docs. They're already linked and this information doesn't seem relevant to people using SinOsc.